website/versioned_docs/version-0.10.0/installation.mdx

```diff
@@ -455,7 +455,7 @@ brew install weaveworks/tap/gitops-ee
 ```
 
 ```
-curl -O https://artifacts.wge.dev.weave.works/releases/bin/0.9.5/gitops-linux-x86_64.tar.gz
+curl -O https://artifacts.wge.dev.weave.works/releases/bin/0.9.5/gitops-$(uname)-$(uname -m).tar.gz
```

Comment:
This doesn't work as `uname` on Linux returns `Linux` but the binary is called `linux`. I'd suppose the same is true for Darwin.

Reply:
@makkes the same is true for Mac at least, where the uploaded binary is gitops-Darwin[...]